### PR TITLE
cleanup (bash-completion)

### PR DESCRIPTION
**Pull request by sboosali**
`new-install`: only complete for local packages.

### CODE REVIEW

File: cabal-install/bash-completion/cabal [hunk 1]
```diff
@@ -137,7 +151,8 @@ function _cabal_targets ()
         [ "$Completion" == new-bench   ] && _cabal_list_targets "benchmark"  && break
         [ "$Completion" == new-haddock ] && _cabal_list_targets              && break
 
-        [ "$Completion" == new-install ] && _cabal_list_external_packages && break
+        [ "$Completion" == new-install ] && _cabal_list_targets "executable" && break
```
Comment (23Skidoo) [inline cabal-install/bash-completion/cabal]: BTW, it should support `v2-` and `v1-` synonyms now too.

**Comment by 23Skidoo** [comment]
OK, feel free to merge.

**Comment by sboosali** [comment]
oh, didn't know about the synonyms, i'll add them

**Comment by 23Skidoo** [comment]
Note that `v2-` (aka `new-`) commands will be the default in 3.0. I'm not sure whether this will require any adjustments.

**Comment by 23Skidoo** [comment]
I'll merge this now, please create a separate PR for further improvements.

**Comment by sboosali** [comment]
sounds good

**Pull request merged**